ZeroMQ.mono.csproj

```diff
@@ -112,9 +112,6 @@
     <Content Include="amd64\libsodium.dll">
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </Content>
-    <Content Include="amd64\libsodium.so">
```

Comment:
These changes also need to be done in the ZeroMQ.VS.csproj file.

Reply:
YA :smile:

Reply:
I think it would be great if you fixed that, since you broke the master by merging the PR prematurely ;)

Reply:
Will be fixed by #124